LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Mac/MACCommandJsonConverter.cs

```diff
@@ -73,16 +75,39 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
                     case Cid.One:
                     case Cid.LinkCheckCmd:
                     case Cid.LinkADRCmd:
+                    {
+                        if (!IsValidLindADRCmd(item, out var errors))
```

Comment:
Instead of this validation method why don't we use tryget and fail if it doesn't work?

Reply:
Done in e9bd89bcf384c5592154f7eb6672c7af465c8264.